asapdiscovery-data/asapdiscovery/data/utils.py

```diff
@@ -23,8 +23,12 @@
 MOONSHOT_CDD_ID_REGEX = r"[A-Z]{3}-[A-Z]{3}-[0-9a-z]+-[0-9]+"
 MPRO_ID_REGEX = r"Mpro-.*?_[0-9][A-Z]"
 
+# Regex patterns that match chains as well, but only capture the main part
```

Comment:
I would say these need more docs on the expected format and what exactly we are trying to capture. Might be good to add for the above  regex as well.